orc8r/cloud/helm/orc8r/charts/secrets/values.yaml

```diff
@@ -54,10 +54,15 @@ secret:
         profile: "prometheus"
 
         prometheusQueryAddress: "http://orc8r-prometheus:9090"
+        prometheusPushAddresses: "http://orc8r-prometheus-cache:9091/metrics"
```

Comment:
This doesn't need to be in metricsd anymore. It's used in orchestrator.yml

Reply:
ack. Removing it.